docker/oso-f22-host-monitoring/Dockerfile

```diff
@@ -38,6 +38,11 @@ ADD pmlogger_control /etc/pcp/pmlogger/control
 ADD pmdas/ /var/lib/pcp/pmdas/
 
 #
+# script to watch health of pmcd
+ADD check-pmcd-status.sh /usr/local/bin/check-pmcd-status.sh
+RUN chmod +x /usr/local/bin/check-pmcd-status.sh
```

Comment:
This shouldn't be needed as long as it's executable in the filesystem at least AFAIK.